samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/README.txt

```diff
@@ -1 +1,40 @@
-﻿Test image is from pexels.com: https://www.pexels.com/photo/red-stop-sign-39080/
\ No newline at end of file
+﻿---
+page_type: sample
+name: "ML.NET Model Builder Stop Sign Object Detection"
+description: "Use ML.NET Model Builder to train a deep learning object detection model in Azure."
+urlFragment: "mlnet-object-detection-model-builder"
+languages:
+- csharp
+products:
+- dotnet
+- dotnet-core
+- ml-dotnet
```

Comment:
I'm not sure if `ml-dotnet` is a valid value here.

In <https://docs.microsoft.com/samples/browse/>, the products shown at the left side doesn't contain ML. You may want to have an internal ask for a node under .NET for ML?

Reply:
Removed ml-dotnet and added visual-studio in latest commit